rootfs/etc/mkinitcpio.conf

```diff
@@ -0,0 +1,6 @@
+# vim:set ft=sh
+
+MODULES=(i915 dm_mod ext4 sha256 sha512 overlay)
```

Comment:
why add the `i915` module here?

Reply:
Sorry, it wasn't supposed to be in here. I wanted to do the "flicker free boot experience" like it is documented here: [Arch Wiki: Plymouth](https://wiki.archlinux.org/index.php/Plymouth#The_plymouth_hook) And as I understand it, these modules are only optional and the fallback image - which is created by default, but you are not using - contains all available modules. So i figured it won't hurt. But you are obviously free to remove it :grin: